scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -7,5 +7,5 @@
  * We are allowing a variance of -5 on `MIN_SIZE` and +5 on `MAX_SIZE` to avoid the need for frequent changes, as bundle sizes can fluctuate
  */
 
-export const MIN_SIZE = 673 - 5;
+export const MIN_SIZE = 759 - 5;
```

Comment:
Are we concerned about this jump in minimum bundle size? Is it due to the polyfill, or just the extra lines of code from the view tracker?

Reply:
I think its actually jumped outside of this branch. I _think_ its either Optimizely related or something to do with the Top Bar OJ component as its imported into the Header, so probably importing a lot of sub-components.